docs/local-library-setup.md

```diff
@@ -0,0 +1,76 @@
+---
+id: local-library-setup
+title: Local libraries setup
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+The local library is created outside of the `android/` and `ios/` folders and makes use of autolinking to integrate with your app. To create local library we will use [create-react-native-library](https://callstack.github.io/react-native-builder-bob/create). This tool contains all the necessary templates.
```

Comment:
"The local library is created outside of..."

That's not the best way to start this page. Could we explain what are "local libraries" first rather than explaining the 'how' first?

Reply:
https://github.com/facebook/react-native-website/pull/3877/commits/642343b6bfee156b8cabe3d1ea3c7e4c9b788960